src/Kiota.Builder/Writers/Ruby/CodeClassDeclarationWriter.cs

```diff
@@ -27,6 +27,9 @@ public override void WriteCodeElement(CodeClass.Declaration codeElement, Languag
                 writer.WriteLine($"module {NormalizeNameSpaceName(codeElement.Parent.Parent.Name)}");
                 writer.IncreaseIndent();
             }
+            if (codeElement?.Inherits?.Name.ToFirstCharacterUpperCase() == "Entity"){
+                codeElement.Inherits.Name = "Graphrubyv4::Utilities::Users::Entity";
```

Comment:
does all the inheritance need to include the fully qualified name of the class? In which case shouldn't this do it more than for just entity?

In the case it's needed only for entity, any change we could leverage a refiner instead? And read the prefix from the configuration?
https://github.com/microsoft/kiota/blob/709ec58d11622d40bcc52aa1954517a31b580c49/src/Kiota.Builder/Refiners/CommonLanguageRefiner.cs#L170